dashbuilder/dashbuilder-runtime/src/main/java/org/dashbuilder/client/screens/view/RuntimeScreenView.html

```diff
@@ -0,0 +1,31 @@
+<!--
+  Copyright 2020 Red Hat, Inc. and/or its affiliates.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<div id="runtimePage">
+    <div class="runtimePageBody">
+        <div
+            class="runtimePage__content runtimeContent--bg-image runtime-blank-slate">
+            <div
+                class="container-fluid runtime-container-fluid--blank-slate">
+                <div class="blank-slate-pf row">
+                    <h1 data-i18n-key="title">Welcome to Dashbuilder Runtimes</h1>
```

Comment:
```suggestion
                    <h1 data-i18n-key="title"></h1>
```
Since you have an i18n-key binding, you don't need the hardcoded string.